starter-code/css/main.css

```diff
@@ -1,10 +1,106 @@
-/*
-Colors to use:
-dark blue: #06365f
-light blue: #1c5380
-*/
+@import url('https://fonts.googleapis.com/css?family=Open+Sans: 200,300,400,500');
 
 body {
-  font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  height:100vh;
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  background-image: url("../images/bgtile.png");
+  background-repeat: repeat-x;
+  background-size: auto 230px;
+  font-family: "Open Sans", sans-serif;
 }
+
+main{
+	height: 80%;
+	width: 100%;
+	margin: auto;
+}
+
+section{
+	display: flex;
+	justify-content: center;
+	padding: 0 10%;
+}
+
+header {
+	padding: 0 1%;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+}
+
+
+footer a {
+	text-decoration: none;
+	color: #1c5380;
+	font-size: 0.8em;
+}
+
+#cp {
+	color: #B4B4B4;
+}
+
+footer {
+	display: flex;
+	font-weight: 800;
+	width: 70%;
+	justify-content: space-evenly;
+	margin: 0 auto; 
+	padding-top: 60px;
+	align-items: flex-bottom;
+}
+
```

Comment:
I would have added a "font-weight: 100" to your "p" tags to make your text a little bit more tinier.